src/index.ts

```diff
@@ -119,6 +136,7 @@ program
     .requiredOption('-i, --identityFile <path-to-identity-file>')
     .requiredOption('-t, --type <type-of-channel>')
     .requiredOption('-s, --source <source-of-channel>')
+    .option('-o, --outputFile <File where write the new Identity>')
```

Comment:
in the other commands you set it as required. maybe here too?

Reply:
right